src/components/ListModal/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Modal from 'react-modal';
+import Card from "../Card"
+
+import './style.css';
+
+class ListModal extends Component {
+  render() {
+    return (
+      <Modal isOpen={this.props.cards.size > 0} contentLabel="Graveyard">
```

Comment:
I like this way of checking whether to open the modal. I should change the card modal to something like this.